test-three/test-three/src/Account/ContactDetails.js

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
+
+class ContactDetails extends Component {
+  render() {
+    return (
+      <div color="red">
```

Comment:
Is adding style like this a good idea?

Reply:
Oh no I was meant to delete that, should never style in the tag, my bad